Xamarin.Forms.Controls/GalleryPages/PlatformSpecificsGalleries/ModalFormSheetPageiOS.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Xamarin.Forms.PlatformConfiguration;
+using Xamarin.Forms.PlatformConfiguration.iOSSpecific;
+
+namespace Xamarin.Forms.Controls.GalleryPages.PlatformSpecificsGalleries
+{
+	public class ModalFormSheetPageiOS : ContentPage
```

Comment:
Can this page have a button that will let the user go back to the Control Gallery? If you click on other platform specific gallery pages you'll notice each one has a button for "Back to gallery" otherwise the user is just kind of stuck

Reply:
Ah of course, was too quick on the actual code, forgot to tidy this one up

Reply:
Done